docs/modules/ROOT/pages/jakartaee-certification/5.2020.6/tck-results-full-5.2020.6.adoc

```diff
@@ -0,0 +1,753 @@
+SHA-256: `b679bac9b1057df894753892a880ba6ade530607dd811157106ed767aa26481f`
+
+** Jakarta Contexts and Dependency Injection 2.0 TCK
++
+See test results in section `cdi`. + 
+Download URL & SHA-256: 
+https://download.eclipse.org/jakartaee/cdi/2.0/cdi-tck-2.0.6-dist.zip[cdi-tck-2.0.6-dist.zip],
+SHA-256:  `75e969a7a3b3c77332154a2008309aad821a923d8684139242048a7640762808`
+** Jakarta Bean Validation 2.0 TCK, 
++
+See test results in section `beanvalidation`. +
+https://download.eclipse.org/jakartaee/bean-validation/2.0/beanvalidation-tck-dist-2.0.5.zip[beanvalidation-tck-dist-2.0.5.zip], 
+SHA-256: `b6778914f78bfcce5d6934347d71502603b1b0a6bbfdfbcf956271c367d40974`
+** Jakarta Batch 1.0 TCK tests are included in Jakarta EE 8 CTS tests, See test results in section `batch` 
+- Java runtime used to run the implementation:
++
+`OpenJDK Runtime Environment (build 1.8.0_242-8u242-b08-0ubuntu3~18.04-b08)
+- Summary of the information for the certification environment:
++
+Apache Derby, Linux, Ubuntu 18.04.3 +
+Interoperatibility tests carried out against Eclipse Glassfish 5.1.0
+
+== Test results
+### appclient 
+
+```
+   [runcts] OUT => [javatest.batch] Completed running 50 tests.
+   [runcts] OUT => [javatest.batch] Number of Tests Passed      = 50
+   [runcts] OUT => [javatest.batch] Number of Tests Failed      = 0
+   [runcts] OUT => [javatest.batch] Number of Tests with Errors = 0
+```
+
+### assembly 
+
+```
+   [runcts] OUT => [javatest.batch] Completed running 30 tests.
+   [runcts] OUT => [javatest.batch] Number of Tests Passed      = 30
+   [runcts] OUT => [javatest.batch] Number of Tests Failed      = 0
+   [runcts] OUT => [javatest.batch] Number of Tests with Errors = 0
+```
+
+### beanvalidation
+
+```
+[INFO] Tests run: 1043, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 657.577 s - in TestSuite
+[INFO] 
+[INFO] Results:
+[INFO] 
+[INFO] Tests run: 1043, Failures: 0, Errors: 0, Skipped: 0
+[INFO] 
+[INFO] 
+[INFO] --- maven-surefire-report-plugin:2.21.0:report-only (generate-test-report) @ hibernate-validator-tck-runner ---
+[WARNING] Unable to locate Test Source XRef to link to - DISABLED
+[INFO] ------------------------------------------------------------------------
+[INFO] BUILD SUCCESS
+```
+### cdi
+
+```
+ [mvn.test] [INFO] Tests run: 1805, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1,818.265 s - in TestSuite
+ [mvn.test] [INFO] 
+ [mvn.test] [INFO] Results:
+ [mvn.test] [INFO] 
+ [mvn.test] [INFO] Tests run: 1805, Failures: 0, Errors: 0, Skipped: 0
+ [mvn.test] [INFO] 
+ [mvn.test] [INFO] 
+ [mvn.test] [INFO] --- maven-surefire-report-plugin:2.19.1:report-only (generate-test-report) @ weld-payara-runner-tck11 ---
+ [mvn.test] [WARNING] Unable to locate Test Source XRef to link to - DISABLED
+ [mvn.test] [INFO] ------------------------------------------------------------------------
+ [mvn.test] [INFO] BUILD SUCCESS
```

Comment:
Trim this bit out - 102 to line 97 can be removed

Reply:
hmm, didn't spot that. Seems porting packages output differently